### PR TITLE
Add product page lifetime warranty section

### LINKED ISSUE #782: Add product page lifetime guarantee section
Screenshot 2022-10-06 at 19 00 47

### PR DESCRIPTION
**Pull request by dhmacs**
closes #782 

## QA

1. Visit the Vercel preview
2. Go to a [product page](https://react-commerce-git-add-product-page-lifetime-warr-d494c2-ifixit.vercel.app/Products/pro-tech-toolkit)
3. If the selected variant has lifetime guarantee it should display a lifetime guarantee section at the bottom of the page
4. Verify that the section is implemented according to mockup and is responsive

### CODE REVIEW

File: frontend/tsconfig.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
          "@models/*": ["models/*"],
          "@components/*": ["components/*"],
          "@templates/*": ["templates/*"],
-         "@layouts/*": ["layouts/*"]
+         "@layouts/*": ["layouts/*"],
+         "@public/*": ["public/*"]
```
Comment (danielcliu-ifixit) [inline frontend/tsconfig.json:14]: What does this change do for this pull?

**Comment by danielcliu-ifixit** [comment]
CR :v: code looks good

**Comment by k0rvusk0r4x** [comment]
- The component is there.
- It looks like the mockup.
- The component is responsive.

QA 😺 

I wasn't able to find the mockup for the mobile version, but most components with a heavy amount of graphics (as opposed to all text) are centered on mobile while the text for this component is left justified.

deploy_block 😿 to double check that we expect the text on the mobile version to be left justified.

**Comment by dhmacs** [comment]
@k0rvusk0r4x here's the link to the mobile version

https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit---Working-files?node-id=6657%3A90027

Screenshot 2022-10-07 at 19 16 35

**Review by masonmcelvain — APPROVED** [review]
un_deploy_block ✌🏻 Looks like the figma

**Pull request merged**